base/poco/Foundation/include/Poco/Platform.h

```diff
@@ -206,8 +208,16 @@
 #elif defined(__AARCH64EB__)
 #    define POCO_ARCH POCO_ARCH_AARCH64
 #    define POCO_ARCH_BIG_ENDIAN 1
-#elif defined(__riscv) && (__riscv_xlen == 64)
-#    define POCO_ARCH POCO_ARCH_RISCV64
+#elif defined(__riscv)
+	#if (__riscv_xlen == 64)
+		#define POCO_ARCH POCO_ARCH_RISCV64
+		#define POCO_ARCH_LITTLE_ENDIAN 1
+	#elif(__riscv_xlen == 32)
```

Comment:
Same here, why are defines for RISC-V 32bit defined? Do we ever intend to support it?

Reply:
I wanted to be consistent with [poco's upstream](https://github.com/pocoproject/poco/blob/devel/Foundation/include/Poco/Platform.h#L138), which doesn't seem necessary right now.

Reply:
Same reason as above.